flink-filesystems/flink-s3-fs-presto/src/test/java/org/apache/flink/fs/s3presto/PrestoS3FileSystemITCase.java

```diff
@@ -48,20 +44,12 @@
  * href="https://docs.aws.amazon.com/AmazonS3/latest/dev/Introduction.html#ConsistencyModel">consistency
  * guarantees</a> and what the {@link com.facebook.presto.hive.s3.PrestoS3FileSystem} offers.
  */
-@RunWith(Parameterized.class)
-public class PrestoS3FileSystemITCase extends AbstractHadoopFileSystemITTest {
-
-    @Parameterized.Parameter public String scheme;
-
-    @Parameterized.Parameters(name = "Scheme = {0}")
-    public static List<String> parameters() {
-        return Arrays.asList("s3", "s3p");
-    }
+class PrestoS3FileSystemITCase extends AbstractHadoopFileSystemITTest {
```

Comment:
Will we lose parameter tests? Additionally, can we check if the number of tests is consistent with the master branch?

Reply:
The parameter "schema" wasn't used in the test before, therefore nothing should change by removing it. 

Let me compare the number of tests and then comment here.